### PR TITLE
#fix floating issue, delegations sort issue, password validation in keystore generation.

### PR DESCRIPTION
**Pull request by blackpanther1881**

### CODE REVIEW

File: src/containers/SignIn/KeyStoreLogin.js [hunk 1]
```diff
@@ -33,40 +33,44 @@ const KeyStoreLogin = (props) => {
...
-            fileReader.readAsText(event.target.uploadFile.files[0], "UTF-8");
-            fileReader.onload = async event => {
-                const res = JSON.parse(event.target.result);
-                const decryptedData = helper.decryptStore(res, password);
-                if (decryptedData.error != null) {
-                    setErrorMessage(decryptedData.error);
-                } else {
-                    let mnemonic = helper.mnemonicTrim(decryptedData.mnemonic);
-                    localStorage.setItem('encryptedMnemonic', event.target.result);
+            if(helper.passwordValidation(password)) {
```
Comment (puneet2019) [inline src/containers/SignIn/KeyStoreLogin.js:36]: will this cause old users to be unable to use Keystore file?

Comment (blackpanther1881) [reply src/containers/SignIn/KeyStoreLogin.js:36]: if they set password to less than 3 characters.

File: src/containers/Staking/Validators/DelegatedValidators/index.js [hunk 2]
```diff
@@ -37,8 +38,9 @@ const DelegatedValidators = (props) => {
         options: {
             sortCompare: (order) => {
                 return (obj1, obj2) => {
-                    let val1 = parseInt(obj1.data.props.children[0]);
-                    let val2 = parseInt(obj2.data.props.children[0]);
+                    console.log(obj1,obj2,"raju");
```
Comment (puneet2019) [inline src/containers/Staking/Validators/DelegatedValidators/index.js:41]: console log..

File: src/containers/SignIn/KeyStoreLogin.js (hunk 1 above)
Comment (puneet2019) [reply src/containers/SignIn/KeyStoreLogin.js:36]: i think we should let old users be able to login with whatever password they created, 
if we want to go forward with this, we have to give a new button to migrate from old keystore to newkeystore, 
i.e. change password.

I know a few people who have forgot mnemonics and depend on keystore file only.

**Review by persistenceAdmin — APPROVED** [review]

**Pull request merged**